### DISCUSSION TITLE
Supreme Concentration

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#70] [Card Balance]
Not even remotely useful.
Original card has same effect as Raging Might and Monkey's Foot. Go for it.

Supreme Concentration (MISHIMA) 
KI 
❌ 1 Action: +1 [A] (until beginning of your next turn)
➡️ Gain opposing warrior's S during combat

**Comment by david-roxygen** [comment]
Totally agree with this suggestion. In this way there is a power for each stat.
